compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEAttrs.td

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_COMPILER_AMDAIE_DIALECT_IREEAMDAIEATTRS
+#define IREE_COMPILER_AMDAIE_DIALECT_IREEAMDAIEATTRS
+
+include "iree-amd-aie/IR/AMDAIEDialect.td"
+include "mlir/IR/AttrTypeBase.td"
+include "mlir/IR/EnumAttr.td"
+
+def AMDAIE_MemSpace_L1 : I32EnumAttrCase<"L1", 1, "l1">;
+def AMDAIE_MemSpace_L2 : I32EnumAttrCase<"L2", 2, "l2">;
```

Comment:
This should be pointed out earlier. I think here "1" and "2" are used for bufferize to allocation, and should match the annotation in AIR. In AIR, "1" is for shared memory and "2" is for local memory. So it's not correct to put 1 as L1 and 2 as L2.